code/game/jobs/job/support.dm

```diff
@@ -420,6 +420,8 @@
 	r_pocket = /obj/item/barcodescanner
 	l_hand = /obj/item/storage/bag/books
 	pda = /obj/item/pda/librarian
+	backpack_contents = list(
+		/obj/item/videocam = 1)
```

Comment:
I would put this bracket on a newline so it fits in with other instances of `backpack_contents`